package/cloudshell/iac/terraform/services/tf_proc_exec.py

```diff
@@ -227,9 +227,8 @@ def _set_service_status(self, status: str, description: str):
             description
         )
 
-    def _init_backend_config(self) -> str:
-        backend_attribute_name = \
-            f"{self._shell_helper.tf_service.cloudshell_model_name}.{ATTRIBUTE_NAMES.REMOTE_STATE_PROVIDER}"
+    def _init_backend_config(self):
+        backend_attribute_name = f"{self._shell_helper.tf_service.cloudshell_model_name}.Remote State Provider"
```

Comment:
why did you remove the use of constant for the attribute name?

Reply:
don`t understand how it got to this state...I was the one who defined this :ATTRIBUTE_NAMES.REMOTE_STATE_PROVIDER

Reply:
fix in commit 16b09f2f1bc270ba7499e05e48d37dee3a4d57e6